main/opcodes/stack-operations.zkasm

```diff
@@ -127,16 +127,17 @@ opPUSH32:
 opAuxPUSHA:
     ; checks zk-counters
     %MAX_CNT_STEPS - STEP - 100 :JMPN(outOfCountersStep)
+    ; check out-of-gas
+    GAS - %GAS_FASTEST_STEP => GAS      :JMPN(outOfGas)
 
     $ => B              :MLOAD(bytecodeLength)
     PC + D - B          :JMPN(opAuxPUSHA2)
-    B - PC => D
+    B - PC => D,E       :JMPZ(lastOpcodeIsPush)
 
 opAuxPUSHA2:
-    ; check out-of-gas
-    GAS - %GAS_FASTEST_STEP => GAS      :JMPN(outOfGas)
     ; read bytes from calldata
     D => E              :CALL(readPush); in: [D,E: bytes to read] out: [E: value read]
```

Comment:
D => E              :CALL(readPush); in: [D,E: bytes to read] out: [E: value read]

Change comment, only in D, E it's just output

Reply:
Both are used at readPush as input

Reply:
true ^^